ims-apple/IMS Apple/Views/Users/UserRegistrationView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  UserRegistrationView.swift
+//  IMS Apple
+//
+//  Created by Diana Zeledon on 30/7/24.
+//
+
+import SwiftUI
+
+struct UserModel: Hashable, Identifiable {
+    var id = UUID()
+    let image: String
+    let name: String
+    let email: String
+    let rol: String
+    let fecha: String
```

Comment:
Can you rename this variable pls?

```suggestion
    let date: String
```